cluster/config.go

```diff
@@ -87,6 +87,17 @@ func (c *ContainerConfig) extractExprs(key string) []string {
 	return exprs
 }
 
+// SwarmID extracts the Swarm ID from the Config.
+// May return an empty string if not set.
+func (c *ContainerConfig) SwarmID() string {
+	return c.Labels[namespace+".id"]
```

Comment:
yes I need it too for another PR, I think we could do a `consts.go` in `/cluster` and export the variable there.